module/zfs/dsl_pool.c

```diff
@@ -1167,7 +1176,9 @@ EXPORT_SYMBOL(dsl_pool_config_enter);
 EXPORT_SYMBOL(dsl_pool_config_exit);
 
 /* BEGIN CSTYLED */
-/* zfs_dirty_data_max_percent only applied at module load in arc_init(). */
+/* zfs_dirty_data_max_percent only applied at module load in arc_init().
+ * or when there is a memory hot plug event
```

Comment:
`arc_init(), or when there is a memory hot plug event.` ?

Reply:
cleaned it up